src/App.tsx

```diff
@@ -1,61 +1,126 @@
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Users } from './types/Users';
+import { Todos } from './types/Todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(userId: number): Users | null {
```

Comment:
It can't be `Users`, bc your func name is `getUserById`
```suggestion
function getUserById(userId: number): User | null {
```